clients/privacy-center/public/fides-js-components-demo.html

```diff
@@ -170,34 +176,40 @@ <h2>Consent Options</h2>
         return;
       }
 
-      // Pretty-print the fides consent object and add it to the page.
-      document.getElementById("consent-json").textContent = JSON.stringify(
-        Fides.consent,
-        null,
-        2
-      );
-
-      // Pretty-print the fides experience config object and add it to the page.
-      document.getElementById("consent-experience").textContent =
-        JSON.stringify(Fides.experience, null, 2);
-
-      // Pretty-print the fides geolocation object and add it to the page.
-      document.getElementById("consent-geolocation").textContent =
-        JSON.stringify(Fides.geolocation, null, 2);
-
-      // Pretty-print the fides options object and add it to the page.
-      document.getElementById("consent-options").textContent = JSON.stringify(
-        Fides.options,
-        null,
-        2
-      );
-
-      // Test behavior of integrations that can be configured without/before their platform scripts.
-      Fides.gtm();
-      Fides.meta({
-        consent: Fides.consent.tracking,
-        dataUse: Fides.consent.data_sales,
-      });
+      // Ensure we only render the consent config to the DOM once Fides is fully initialized
+      let timerId = setInterval(() => {
```

Comment:
Can you add a `DEFER: ...` for the ticket related to the JS events API? We should tidy up this timeout when that is ready